docs/development-api/github.md

```diff
@@ -20,6 +20,52 @@ const buildName = GitHub.buildNameFromEnvironment();
 
 ---
 
+## commitAndPush
+
+Commits and pushes file changes from the local workspace to a specified GitHub branch.
+
+Commits using this method appear verified on GitHub.
+
+```typescript
+import { GitHub } from 'skuba';
+
+const commitId = await GitHub.commitAndPush({
+  dir: './',
+  branch: 'some-branch',
+  fileChanges: {
+    additions: [{ contents: '', path: 'another-path' }],
+    deletions: [{ path: 'some-path' }],
+  },
+  messageHeadline: 'some-commit',
+  messageBody: 'extra-body',
+  updateLocal: true, // Updates the local git repository to reflect the new remote branch state
+});
+```
+
+---
+
+## commitAndPushAllChanges
+
+Retrieves all file changes from the local git repository using [getChangedFiles], commits and pushes file changes from the local workspace to a specified GitHub branch.
+
+Commits using this method appear verified on GitHub.
+
+`undefined` is returned instead of a commit id when there are no changes to push.
```

Comment:
```suggestion
Retrieves all file changes from the local Git repository using [getChangedFiles],
then commits and pushes the changes to a specified GitHub branch using [commitAndPush](#commitandpush).

Commits pushed using this method appear verified on GitHub.

`undefined` is returned instead of a commit ID when there are no changes to push.
```